protected/humhub/modules/comment/views/comment/edit.php

```diff
@@ -29,7 +29,10 @@
         <?= $form->field($comment, 'message')->widget(RichTextField::class, [
             'id' => 'comment_input_' . $comment->id,
             'layout' => RichTextField::LAYOUT_INLINE,
-            'pluginOptions' => ['maxHeight' => '300px'],
+            'pluginOptions' => [
+                'maxHeight' => '300px',
+                'markdownEditorMode' => Yii::$app->user->getIdentity()->settings->get('markdownEditorMode'),
```

Comment:
Instead of passing the `markdownEditorMode` per instance (comment, post, e.g.) I would prefer a more global approach.
 
We use `View::registerJsConfig` to pass configurations or translations to the JS part on full page loads. What do you think about using this also for the editor mode?

Reply:
Done